include/engine/api/trip_parameters.hpp

```diff
@@ -47,7 +47,27 @@ namespace api
  */
 struct TripParameters : public RouteParameters
 {
-    // bool IsValid() const; Falls back to base class
+    TripParameters()
+        : RouteParameters(false,
+                          false,
+                          false,
+                          RouteParameters::GeometriesType::Polyline,
+                          RouteParameters::OverviewType::Simplified,
+                          {}),
```

Comment:
Why setting parameters here for RouteParameters?

Can't this be `TripParameters() = default;` and setting `source = -1` and `destination = -1` below?